src/utils/custom_evaluation.py

```diff
@@ -479,6 +577,23 @@ def evaluate_policy_meta_agent(
         if render:
             env.render()
 
+    test_path = os.path.join(os.path.dirname(__file__),'..','..','agent_data')
+
+    if not os.path.isdir(test_path):
+        print("test no dir")
+        os.mkdir(test_path)
+
+    dir_path = os.path.join(os.path.dirname(__file__),'..','..','agent_data', 'agent')
```

Comment:
```suggestion
    dir_path = os.path.join(os.path.dirname(__file__),'..','..','agent_data', 'agent')

    if not os.path.isdir(dir_path):
        print("Creating directory for evaluation files of agent")
        os.mkdir(dir_path)

```

Vielleicht verstehe ich was falsch, aber reicht das nicht so? Wofür der `test_path` und muss man nicht auch noch überprüfen, ob der Ordner `agent` existiert?